content/events/2023-new-york-city/welcome.md

```diff
@@ -0,0 +1,97 @@
++++
+
+<div class="row">
+  <div class="col-lg-5 my-4">
+    <img alt="DevOpsDays NYC 2023" src="/events/2023-new-york-city/logo.png" style="display: block; margin: auto; max-width: 400px">
+  </div>
+  <div class="col-lg-7 my-4">
+    <div class = "row">
+      <div class = "col-md-1 mx-5 px-0">
+        <h4 class="my-3"><strong>Dates</strong></h4>
+      </div>
+      <div class = "col-md-9">
+        <h4 class="my-3"><small>{{< event_start >}} - {{< event_end >}}</small></h4>
+      </div>
+    </div>
+    <div class = "row">
+      <div class = "col-md-1 mx-5 px-0">
+        <h4 class="my-3"><strong>Program</strong></h4>
+      </div>
+      <div class = "col-md-9">
+        <h4 class="my-3"><small>View the {{< event_link page="program" text="Event Schedule." >}}</small></h4>
+      </div>
+    </div>
+    <!-- <div class = "row">
+      <div class = "col-md-1 mx-5 px-0">
+        <h4 class="my-3"><strong>Open Space</strong></h4>
+      </div>
+      <div class = "col-md-9">
+        <h4 class="my-3"><small>View the open space schedule <a href="" target="_blank">here</a></small></h4>
+      </div>
+    </div> -->
+    <div class = "row">
+      <div class = "col-md-1 mx-5 px-0">
+        <h4 class="my-3"><strong>Location</strong></h4>
+      </div>
+      <div class = "col-md-9">
+        <h4 class="my-3"><small>{{< event_location >}}
+      </div>
+    </div>
+    <div class = "row">
+      <div class = "col-md-1 mx-5 px-0">
+        <h4 class="my-3"><strong>Register</strong></h4>
+      </div>
+      <div class = "col-md-9">
+        <h4 class="my-3"><small>{{< event_link page="registration" text="Tickets are on sale now!" >}}</small></h4>
+      </div>
+    </div>
+    <!-- <div class = "row">
+      <div class = "col-md-1 mx-5 px-0">
+        <h4 class="my-3"><strong>Propose</strong></h4>
+      </div>
+      <div class = "col-md-9">
+        <h4 class="my-3"><small>Our CFP is now closed. Sorry, we can't accept late submissions.</small></h4>
+      </div>
+    </div> -->
+    <div class = "row">
+      <div class = "col-md-1 mx-5 px-0">
+        <h4 class="my-3"><strong>Sponsors</strong></h4>
+      </div>
+      <div class = "col-md-9">
+        <h4 class="my-3"><small>{{< event_link page="sponsor" text="Sponsor the conference!" >}}</small></h4>
+      </div>
+    </div>
+    <div class = "row">
+      <div class = "col-md-1 mx-5 px-0">
+        <h4 class="my-3"><strong>Contact</strong></h4>
+      </div>
+      <div class = "col-md-9">
+        <h4 class="my-3"><small>{{< event_link page="contact" text="Get in touch with the organizers" >}}</small></h4>
+      </div>
+    </div>
+    <div class = "row">
+      <div class = "col-md-1 mx-5 px-0">
+        <h4 class="my-3"><strong>Slack</strong></h4>
+      </div>
+      <div class = "col-md-9">
+        <h4 class="my-3"><small> Join the <a href="https://launchpass.com/nycdevops" target="_blank">NYC Devops Slack Workspace</a></small></h4>
+      </div>
+    </div>
```

Comment:
I believe we usually leave this disabled until closer to the event, but theres prob no harm in allowing people to join now if we prefer that. We might jump on there and moderate spam better though, its kind of a mess 😅